lbrynet/core/client/ConnectionManager.py

```diff
@@ -213,11 +218,10 @@ def _connect_to_peer(self, peer):
         if peer is None or self.stopped:
             return
 
-        from twisted.internet import reactor
-
         log.debug("%s Trying to connect to %s", self._get_log_name(), peer)
         factory = ClientProtocolFactory(peer, self.rate_limiter, self)
         self._peer_connections[peer] = PeerConnectionHandler(self._primary_request_creators[:],
                                                              factory)
         connection = reactor.connectTCP(peer.host, peer.port, factory)
         self._peer_connections[peer].connection = connection
+        return factory.deferred
```

Comment:
can you document what this returns and when the deferred that is returned gets fired?

It will save whoever is reading this code next from having to dig through a bunch of code.